pkg/lua/patches/0007-Add-a-proper-makefile.patch

```diff
@@ -0,0 +1,358 @@
+From 1cfceae0d4d4ead217af7b66ed24b3909eb22dd2 Mon Sep 17 00:00:00 2001
+From: Juan Carrano <j.carrano@fu-berlin.de>
+Date: Mon, 14 May 2018 15:31:09 +0200
+Subject: [PATCH 7/8] Add a proper makefile.
+
+The new makefile has automatic dependencies, can buid a debug an
+a non debug version simultaneously and is way cleaner.
+---
+ .gitignore |   7 ++
+ Makefile   | 116 +++++++++++++++++++++++++++++++
+ makefile   | 198 -----------------------------------------------------
+ 3 files changed, 123 insertions(+), 198 deletions(-)
+ create mode 100644 .gitignore
+ create mode 100644 Makefile
+ delete mode 100644 makefile
+
+diff --git a/.gitignore b/.gitignore
+new file mode 100644
+index 00000000..08e46c95
+--- /dev/null
++++ b/.gitignore
+@@ -0,0 +1,7 @@
++*.o
++*.gch
++*.d
++*.a
++lua
++lua-dbg
++*.patch
+diff --git a/Makefile b/Makefile
+new file mode 100644
+index 00000000..2735db8d
+--- /dev/null
++++ b/Makefile
+@@ -0,0 +1,116 @@
```

Comment:
Might be worth checking with Gaetan whether these changes are in line with the work he's doing on the build system. In other words, whether they incorporate the work he's doing or do things in a different way

Reply:
This makefile is not being used (it is overwritten). The patch is there because it's useful if one wants to compile the standalone interpreter.

Reply:
ok